webapp/src/app/projects/components/team-overview/team-overview.component.html

```diff
@@ -14,5 +14,14 @@ <h5 class="font-serif m-0" i18n>Project Team</h5>
         (remove)="removeProjectUser(project.id, $event.userId)"
       ></app-team-member>
     </div>
+    <div *ngFor="let invite of (projectInvites$ | async)">
```

Comment:
Maybe we can add a little subtitle or something for hinting to the user that what follows are the invites.

Reply:
This is what I came up with:

<img width="341" alt="Screenshot 2019-04-02 at 17 45 32" src="https://user-images.githubusercontent.com/7799003/55411945-332f9d00-556f-11e9-87aa-1b717f865155.png">

Reply:
Looks good!